metadata.yaml

```diff
@@ -10,8 +10,53 @@ description: |
 requires:
   mimir-worker:
     interface: mimir_worker
+    description: |
+      Coordinator forwards operations data to the workers:
+      - s3 storage information
```

Comment:
and hash ring and config, but yes

Reply:
also, do we need to forward loki/tempo endpoints?

Reply:
[needs discussion]: above

Reply:
Done